knowledge-base/grid-state-filter-nullable-bool.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Programatically filter a nullable bool column in the Grid by null values
+description: Learn how to programatically filter nullable bool values
+type: how-to
+page_title: How to filter a Grid column bound to a nullable bool by the null values programatically
```

Comment:
Try to use up to 50 characters in the `page_title`.

The `page_title` length should be no more than 50 characters because the total [maximum length](https://docs.telerik.com/style-guide/metadata) is 70 characters and 24 of the 70 characters are already consumed by default during build as we automatically append the ` - Telerik UI for Blazor` suffix.

Reply:
If the title `page_title` cannot be reasonably shortened, use the same value as in the `title` filed.